homeassistant/components/octoprint/camera.py

```diff
@@ -0,0 +1,63 @@
+"""Support for monitoring OctoPrint binary sensors."""
+from __future__ import annotations
+
+from pyoctoprintapi import OctoprintClient
+
+from homeassistant.components.mjpeg.camera import MjpegCamera
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import OctoprintDataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the available OctoPrint binary sensors."""
```

Comment:
Not a review, but while glancing at the code I happened to notice this copy paste error.

Reply:
Nice catch, fixed. Thanks!